syntax = "proto3";
package recommand;
import "pb/common/common.proto";

service RecommandService {
    rpc Recommand (RecommandRequest) returns (RecommandReply) {
    }

    rpc SetDefaultRecommandItems (SetDefaultRecommandItemsRequest) returns (EmptyMessage) {
    }
}

message EmptyMessage {
}

message RecommandRequest {
    string user_id = 1;
    uint32 page = 2;
    uint32 page_size = 3;
}

message RecommandReply {
    repeated string item_ids = 1;
    common.PageInfo page_info = 2;
}

message SetDefaultRecommandItemsRequest {
    repeated string item_ids = 1;
}
